### PR TITLE
DOMA-2503 Fix classifier shortening

### PR DESCRIPTION
**Pull request by VKislov**
1. Before : 
image
2. After: 
image

### CODE REVIEW

File: apps/condo/domains/ticket/utils/clientSchema/Renders.tsx [hunk 1]
```diff
@@ -107,7 +107,7 @@ export const getClassifierRender = (intl, search: FilterValue) => {
         const placeClassifier = get(record, ['placeClassifier', 'name'])
         const postfix = `\n(${placeClassifier})`
 
-        return getTableCellRenderer(search, true, postfix, null, POSTFIX_PROPS)(text)
+        return getTableCellRenderer(search, false, postfix, null, POSTFIX_PROPS)(text)
```
Comment (dkoviazin) [inline apps/condo/domains/ticket/utils/clientSchema/Renders.tsx:110]: please, add screenshots 
1. before ...
2. after ...

Comment (VKislov) [reply apps/condo/domains/ticket/utils/clientSchema/Renders.tsx:110]: 1. Before : 
image
2. After: 
image

Comment (pahaz) [reply apps/condo/domains/ticket/utils/clientSchema/Renders.tsx:110]: In a issue description :)

**Comment by pahaz** [comment]
It’s hard to understand what’s going on here

**Comment by VKislov** [comment]
`getTableCellRenderer ` has a function to shorten the string and add an ellipsis to the end. When used in conjunction with the native antd function, they do not work correctly. In `getTableCellRenderer` I passed `false` to use only the native solution from the antd library

**Pull request merged**